covid-dashboard/src/js/modules/mapAddMarkers.js

```diff
@@ -0,0 +1,33 @@
+import countries from './countries.js';
+import toFixed from './toFixed.js';
+
+export const circles = {};
+
+export default (map) => {
+  countries.stats.forEach((country) => {
```

Comment:
[Optional] I believe it would be better to fill countries with shades of colors instead of circle markers to show cases in that country. In the example app circles are used, but they have more detailed locations(like for each state in USA or India). Right now it looks like USA does not have that much cases, while it's leader in a list. Besides markers are bad for small countries, since they overlap the whole country with them.

Reply:
I agree, that shades are best solution, but I haven't time to do that and now also have no time, because I want to make a presentation.